submit_and_compare/static/html/submit_and_compare_edit.html

```diff
@@ -18,7 +18,15 @@
           </div>
           <span class="tip setting-help">This assigns an integer value representing the weight of this problem</span>
         </li>
-        
+
+        <li class="field comp-setting-entry is-set">
```

Comment:
Is it worth converting this XBlock over to using the `xblock-utils` lib instead of doing this by hand?